encryption/src/main/java/org/wildfly/security/encryption/SecretKeyUtil.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
```

Comment:
Just a minor comment but looks like all the copyright versions in files in the encryption module in this commit should be updated to 2021 :)

Reply:
I think these should be Ok, some of this code has actually been living in public repositories since 2019 - all the new stuff written this year has the date set to 2021.